lib/iris/fileformats/pp.py

```diff
@@ -681,10 +681,11 @@ def __init__(self, value):
             The initial value which will determine the flags.
 
         """
+        value = int(value)
```

Comment:
This will improve/fix the error message when passing a string that doesn't look like a number. For bonus points it'd be great to tweak `test_negative` and `test_invalid_str` in `lib/iris/tests/unit/fileformats/pp/test__LBProc.py` to use check the message with `assertRaisesRegexp`.